auto_scan.py

```diff
@@ -478,7 +521,16 @@ def main(args):
 
```

Comment:
Given that we're now trying to dynamically lookup and/or create integrations per-subaccount, would it make sense to wrap the `for subaccount in subaccounts` worker loop in a try/except to continue with additional sub-accounts if one in particular were to fail? Or would you prefer to be one fail, all fail?